### ISSUE TITLE
Modifying original response in middleware

### ISSUE THREAD

**Issue by Jan-Dosedel** [buger/goreplay#650]
Thanks for the Goreplay, it's really great tool. In my use case I would need to modify the original response based on some logic in the middleware. Is there any possibility to change the original response? Although I can modify the original response in the middleware but it does not affect on the output of the original request. It would be really nice feature.

Thanks for reply Jan

**Comment by nagendra547** [comment]
Yes, you can modify the request of original response. However you cannot modify the actual original response sent to end clients/endusers.  Middleware can get once you use the parameter.

  --output-http-track-response
